pkg/bee/client.go

```diff
@@ -750,6 +767,54 @@ func (c *Client) UploadFile(ctx context.Context, f *File, o api.UploadOptions) (
 	return
 }
 
+func (c *Client) UploadActFile(ctx context.Context, f *File, o api.UploadOptions) (err error) {
+	h := fileHasher()
+	r, err := c.api.Act.Upload(ctx, f.Name(), io.TeeReader(f.DataReader(), h), o)
```

Comment:
should we have also some `defer r.Close()`

Reply:
r is ActUploadResponse struct